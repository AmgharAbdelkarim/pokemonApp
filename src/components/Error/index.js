import React from "react";

const Error = () => {
  return <h3>Something Went Wrong</h3>;
};

export default Error;
