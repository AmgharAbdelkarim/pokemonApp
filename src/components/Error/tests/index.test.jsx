import { cleanup, render } from "@testing-library/react";

import Error from "components/Error";

afterEach(cleanup);

describe("EmptyState", () => {
  it("should EmptyState render correctly", () => {
    const { asFragment, getByText } = render(Error());
    expect(getByText("Something Went Wrong")).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
