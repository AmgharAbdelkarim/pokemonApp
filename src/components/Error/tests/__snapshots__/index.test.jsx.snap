// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState should EmptyState render correctly 1`] = `
<DocumentFragment>
  <h3>
    Something Went Wrong
  </h3>
</DocumentFragment>
`;
